virttest/libvirt_vm.py

```diff
@@ -967,6 +967,10 @@ def add_memballoon(help_text, memballoon_model):
         if params.get("virt_install_debug") == "yes":
             virt_install_cmd += " --debug"
 
+        emulator_path = params.get("emulator_path", None)
+        if emulator_path and has_sub_option('boot', 'emulator'):
+            virt_install_cmd += " --boot emulator=%s" % emulator_path
```

Comment:
@PandaWei finding the PR got merged, will send this suggestion as a seperate PR.